app/controllers/prison/email_previews_controller.rb

```diff
@@ -4,7 +4,11 @@ class Prison::EmailPreviewsController < ApplicationController
   include BookingResponseContext
 
   def show
-    render html: email_preview
+    if booking_response.valid?
+      render html: email_preview
+    else
+      render text: booking_response.errors.full_messages
```

Comment:
Ignore, I thought this was non JS, probably worth thinking if we can render a styled view